syntax = "proto3";

package echo_proto;

message EchoReq
{
    string sid = 1;
    int32 loginId = 2;
    string info = 3;

    enum Uri
    {
        PLACEHOLDER = 0;
        URI = 31588; // (123 << 8 | 100);
    }
}

message EchoRsp
{
    int32 retCode = 1;
    string sid = 2;
    int32 loginId = 3;
    string info = 4;

    enum Uri
    {
        PLACEHOLDER = 0;
        URI = 31589; // (123 << 8 | 101);
    }
}


